config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -0,0 +1,280 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file eases your Rails 7.1 framework defaults upgrade.
```

Comment:
Is the plan to merge this and do this upgrade process later?  Does that need a ticket?